schemas/preferences.sublime-settings.json

```diff
@@ -1262,6 +1262,21 @@
           "markdownDescription": "The maximum file size (in bytes) after which syntax highlighting will be disabled for files and will be shown as plain text.",
           "type": "integer",
           "default": 16777216
+        },
+        "find_scroll_highlights_limit": {
+          "markdownDescription": "The maximum number of find results to show in the scroll bar. If this number is exceeded no results will be shown. Use `0` to remove this limit.",
+          "type": "integer",
+          "default": 8192
+        },
+        "find_highlight_matches_max_size": {
+          "markdownDescription": "The maximum file size (in bytes) for which the find option **Highlight matches** will highlight matches (affects non-regexp searches). Use `0` to remove this limit.",
+          "type": "integer",
+          "default": 16777216
+        },
+        "find_regex_highlight_matches_max_size": {
+          "markdownDescription": "The maximum file size (in bytes) for which the find option **Regular expression** will highlight matches (affects regexp searches). Use `0` to remove this limit.",
```

Comment:
My suggestion included more changes because I thought it would be more syntactically correct. IMO both `find_highlight_matches_max_size` and `find_regex_highlight_matches_max_size` should be describing the functionality in terms of the `Highlight matches` option, not `Regular expression`,  since they affects the matches highlighting.

Reply:
Agreed. Sorry about this, looks like I did not read the previous review carefully.